housekeeper/store/filters/version_bundle_filters.py

```diff
@@ -17,7 +17,7 @@ def filter_version_by_name_and_bundle_date(
 class VersionBundleFilters(Enum):
     """Define Version filter functions."""
 
-    FILTER_BY_DATE_AND_NAME: Callable = filter_version_by_name_and_bundle_date
+    BY_DATE_AND_NAME: Callable = filter_version_by_name_and_bundle_date
```

Comment:
definelty out of scope for the renaming. 

Should this filter not be two separate ones? 
`filter_version_by_name`
`filter_version_by_bundle_date`

These can then be stacked to achieve the same goal when building the query, right?

Reply:
Agreed, but in another PR